source/client-side-encryption/client-side-encryption.rst

```diff
@@ -597,6 +631,66 @@ message.
 
 Return "access_token" as the credential.
 
+
+Obtaining an Access Token for Azure Key Vault
+`````````````````````````````````````````````
+
+.. versionadded:: 1.12.0 2022/08/16
+
+Virtual machines running on the Azure platform have one or more *Managed
+Identities* associated with them. From within the VM, an identity can be used by
+obtaining a access token via HTTP from the *Azure Instance Metadata Service*
+(IMDS). `See this documentation for more information`__
+
+__ https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http
+
+.. default-role:: math
+
+If an Azure VM has more than one managed identity, requesting an access token
+requires additional query parameters |--| `objectId`, `clientId`, and `miResId`
+|--| that disambiguate the request. We'll refer to these parameters as `M`. If
```

Comment:
I am unsure if more than one managed identity is a common use-case we need to support. I have asked in [DRIVERS-1780](https://jira.mongodb.org/browse/DRIVERS-1780?focusedCommentId=4761705&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4761705)

Reply:
After offline discussion with produce management, there are no known requests for supporting multiple user-assigned managed identities. We can proceed with not supporting multiple user-assigned managed identities.